### ISSUE TITLE
onInitialise doesn't called before any user action

### ISSUE THREAD

**Issue by dolav** [insites/cookieconsent#474]
In case the user didn't take any action and the library doesn't store the cookie yet, the function onInitialise [doesn't call](https://github.com/insites/cookieconsent/blob/master/src/cookieconsent.js#L693).
This is in contrast to the API doc which specifies that:

> It can be used to tell you if the user has already consented or not as soon as you initialise the tool.

```
var complete = this.options.onInitialise.bind(this);
...
var match = allowed.indexOf(answer) >= 0;
if (match) {
  complete(answer);
}
```

**Comment by relicmelex** [comment]
@dolav good catch, sorry for the delayed reply. I'm PR'ing the fix as this message is posted. I'll tag this ticket in the PR so you can follow with what release it goes out in.

**Comment by inpercima** [comment]
Is there a plan when this will come into the master and when it will be released?
